mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -1159,6 +1159,22 @@ func ResourceContainerCluster() *schema.Resource {
 								},
 							},
 						},
+						"advanced_datapath": {
+							Type:        schema.TypeList,
+							Optional:    true,
+							Computed:    true,
+							MaxItems:    1,
+							Description: `Configuration of Advanced Datapath Observability features.`,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
```

Comment:
Should we add `relay_mode` at the same time? https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#Cluster.AdvancedDatapathObservabilityConfig